### PR TITLE
Add welcome pages 3 and 4

### PR DESCRIPTION
**Pull request by craigcook**
Prioritizing "tier 1" locales. We'll be getting French and German translations in-house. All others are opt-in.

### CODE REVIEW

**Comment by peiying2** [comment]
@craigcook Is welcome/3 due on Nov 20? And welcome/4 on Dec 11? Want to double confirm.

**Comment by craigcook** [comment]
> @craigcook Is welcome/3 due on Nov 20? And welcome/4 on Dec 11? Want to double confirm.

Correct. We're getting FR and DE provided in-house for both, and those are the target launch dates for these pages in those locales. Other locales can opt in at their leisure.

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -287,6 +287,20 @@
             'pt-BR', 'pt-PT', 'ru', 'zh-CN', 'zh-TW',
         ],
     ],
+    'firefox/welcome/page3.lang' => [
+        'priority'          => [
+            1 => ['de', 'fr', 'en-GB', 'en-CA'],
```
Comment (peiying2) [inline app/config/sources.inc.php]: needs a `,` after the last locale entry.

File: app/config/sources.inc.php [hunk 2]
```diff
@@ -287,6 +287,20 @@
...
+        ],
+        'deadline'          => '2019-11-20',
+        'supported_locales' => $mozillaorg,
+    ],
+    'firefox/welcome/page4.lang' => [
+        'priority'          => [
+            1 => ['de', 'fr', 'en-GB', 'en-CA'],
```
Comment (peiying2) [inline app/config/sources.inc.php]: needs a `,` after the last locale entry.

**Comment by peiying2** [comment]
@craigcook sorry, my bad, what you did the first time was correct. The closing bracket is on the same line of the last locale, the comma is not needed. Please revert back to the prior version, then I will merge.

**Comment by craigcook** [comment]
Oh looks like that trailing comma is only needed when there's a line break

**Pull request merged**